assets/js/calypsoify.js

```diff
@@ -0,0 +1,42 @@
+( function( $ ) {
+    'use strict';
+
+    /**
+     * Record checklist task click
+     */
+    $( '.checklist__task-title a, .checklist__task-secondary a' ).click( function() {
+        var $task = $( this ).closest( '.checklist__task' )
+        var status = $task.hasClass( 'is-completed' ) ? 'complete' : 'incomplete';
+        var taskTitle = $task.data('title');
+
+        window.jpTracksAJAX.record_ajax_event(
+            'atomic_wc_tasklist_clicked',
```

Comment:
This should be `atomic_wc_tasklist_click`

> use a singular noun or present tense verb

Reply:
Updated!